memflow-win32/src/ida_signatures.rs

```diff
@@ -0,0 +1,34 @@
+/// Generates a regex string from an ida-like hex string.
+/// Made by [@ConnorBP] :)
```

Comment:
I think we shouldn't put authors into individual function comments. We could rather have a contributors document in the root of the project for this :+1: